Detections/ASimWebSession/SuspiciousUAPowershell.yaml

```diff
@@ -0,0 +1,35 @@
+id: 42436753-9944-4d70-801c-daaa4d19ddd2
+name: Suspicious User-Agent Detected - Windows PowerShell
```

Comment:
Same texts and comments as the above. Would be a single rule - just add the threat category to the lookup table. What do you think?

Reply:
Nice idea but complicated to implement. A simple lookup would not work since we match UA with "_has_" , not "=="

Reply:
You forgot [has_any_index](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/has-any-index-function). Get all the patterns, compare and get the index of the match to lookup the additional info.